client/src/fluid/Fluid.res

```diff
@@ -1,12 +1,15 @@
-/* Specs:
- * Pressing enter
- * https://trello.com/c/nyf3SyA4/1513-handle-pressing-enter-at-the-end-of-a-line
- * https://trello.com/c/27jn8uzF/1507-handle-pressing-enter-at-the-start-of-a-line
- * renaming functions:
- * https://trello.com/c/49TTRcad/973-renaming-functions
- * movement shortcuts:
- * https://trello.com/c/IK9fQZoW/1072-support-ctrl-a-ctrl-e-ctrl-d-ctrl-k
- */
+// Specs:
+// - Pressing enter
+//   - https://trello.com/c/nyf3SyA4/1513-handle-pressing-enter-at-the-end-of-a-line
+//   - https://trello.com/c/27jn8uzF/1507-handle-pressing-enter-at-the-start-of-a-line
+// - renaming functions:
+//   - https://trello.com/c/49TTRcad/973-renaming-functions
+// - movement shortcuts:
+//   - https://trello.com/c/IK9fQZoW/1072-support-ctrl-a-ctrl-e-ctrl-d-ctrl-k
+//
+// WHAT all of these links are inaccessible.
```

Comment:
I intend to move these somewhere accessible. Really we just need to decide where that is.